dev/clint/tests/rules/test_track_api_usage_outermost.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+from clint.config import Config
+from clint.index import SymbolIndex
+from clint.linter import lint_file
+from clint.rules import TrackApiUsageOutermost
+
+
+def test_track_api_usage_outermost_valid(
+    index: SymbolIndex, config: Config, tmp_path: Path
+) -> None:
+    """Test that @track_api_usage as the outermost decorator does not trigger a violation."""
+    tmp_file = tmp_path / "file.py"
+    tmp_file.write_text(
+        """
+from mlflow.telemetry.track import track_api_usage
+
+@track_api_usage
+@property
+def some_function():
+    pass
```

Comment:
is this valid?

Reply:
No it should be the outermost decorator

Reply:
Got it. We need to be careful with decorators that forget to use `functools.wraps`. Here's an example:

```python
def deco(f):
    print("name:", f.__name__)

    def wrapper(*args, **kwargs):
        return f(*args, **kwargs)

    return wrapper


@deco  # -> wrapper
@deco  # -> foo
def foo():
    pass

```

Reply:
What do you mean by `track_api_usage is applied after other decorators`? This decorator shouldn't impact on the signature of the decorated function

Reply:
That's tricky.

Reply:
The API name, we use `f"{func.__module__}.{func.__qualname__}"` as the API name, and this will be changed with above decorator

Reply:
btw can you check if https://github.com/mlflow/mlflow/pull/16522#discussion_r2182275894 really occurs?

Reply:
Yes it's true